vcpkg/get_started/get-started-linux.md

```diff
@@ -0,0 +1,131 @@
+---
+title: Install and use packages on Linux
+description: Tutorial guides the user through the process of installing and using packages on Linux with vcpkg.
+author: JavierMatosD
+ms.date: 08/23/2023
+ms.author: javiermat
+ms.prod: vcpkg
+---
+# Install and use packages on Linux
+
+## Description
+
+This tutorial guides you through setting up a new C++ "Hello World" project using the `fmt` library. You'll manage dependencies, build the application, and eventually run your project, all using vcpkg. This tutorial assumes you have vcpkg setup and ready to use. If not, check out our initial setup [tutorial](setup-vcpkg.md).
+
+### Prerequisites
+
+For this tutorial, you'll need: a terminal, `git`, `cmake`, and a C++ compiler.
+
+## Step 1: Setup the project
+
+### Step 1.1: Create the project directory
+
+Create a directory for your new project next to your vcpkg clone and navigate into it:
+
+```bash
+mkdir helloworld && cd helloworld
+```
+
+### Step 1.2 Create the manifest
+
+From within the `helloworld` directory, run:
+
+```bash
+..\vcpkg\vcpkg new --name helloworld --version 1.0
+```
+
+>[!NOTE]
+>Ensure the project directory name matches exactly with the name you give to the `vcpkg new` command. This will ensure subsequent commands work smoothly.
+
+The `vcpkg new` command will add a `vcpkg.json` file and a `vcpkg-configuration.json` file to your projects directory. The `vcpkg.json` file should look like this:
+
+```json
+{
+    "name": "helloworld",
+    "version": "1.0"
+}
+```
+
+The `vcpkg.json` file serves as the foundation to integrate vcpkg in your C++ project. Initially, it contains your project's name and version; additional attributes like dependencies and supported features can be added as your project grows.
+
+To learn more about `vcpkg.json`, check out our reference [documentation](..\reference\vcpkg-json.md).
+
+### Step 1.3 Remove configuration file
+
+The `vcpkg-configuration.json` file serves a different purpose. This file allows full control over your dependencies' sources and it's out of scope for this tutorial.
+
+For this tutorial, our dependencies all come from the vcpkg registry at <https://github.com/Microsoft/vcpkg>. When there's no explicit configuration provided, vcpkg uses this registry as default. So the next step is to delete this file.
+
+```bash
+rm vcpkg-configuration.json
+```
+
+## Step 2: Add dependencies and project files
+
+Now that the project is properly set up, lets add the `fmt` library as a dependency and generate the project files.
+
+### Step 2.1: Add `fmt` dependency
+
+Since this project will use the `fmt` library to print "Hello World", you'll want to add `fmt` to the "dependencies" list in the manifest. To do this, run:
```

Comment:
```suggestion
First, run the following command to add `fmt` to the `dependencies` list in your vcpkg.json file:
```

Reply:
Why this redundancy? You just said we were going to add the fmt library as a dependency.  Think of your word count as something you have to pay for, because the reader does in terms of time/effort.